JitsiConference.ts

```diff
@@ -575,7 +575,7 @@ export default class JitsiConference extends Listenable {
         this._statsCurrentId = config.statisticsId ?? Settings.callStatsUserName;
         this.room = this._xmpp.createRoom(
             this.options.name, {
-                customDomain: this.options.customDomain,
+                ...config,
```

Comment:
Why this change? I don't like it. I was thinking that generally we would like to avoid passing everything and be more specific and pass only what is needed.

Reply:
That is a fix, as visitors are broken with current code from master. 
I will drop this commit once https://github.com/jitsi/lib-jitsi-meet/pull/2902 is merged and I rebase to it.